app/models/github_model.rb

```diff
@@ -49,7 +52,33 @@ def initialize(client, id_attributes, **options)
         attr[gh_attr.to_sym] = github_response(client, id_attributes.values.compact, options).send(gh_attr)
       end
 
-      remove_instance_variable("@response")
+      local_cached_attributes.each do |gh_attr|
+        define_singleton_method(gh_attr) do |use_cache: true|
+          field_name = "github_#{gh_attr}".to_sym
+
+          no_cache_options = options.dup
+          no_cache_options[:headers] = GitHub::APIHeaders.no_cache_no_store
```

Comment:
We add this to the options hash because if we make an API request, it's either because we want to skip the cache or we need to warm our cache. In both cases, we should ask GitHub for the most recent data.